File: 2148-count-elements-with-strictly-smaller-and-greater-elements/2148-count-elements-with-strictly-smaller-and-greater-elements.cpp
class Solution {
public:
    int countElements(vector<int>& n) {
        int m1=INT_MIN;
        int m2=INT_MAX;
        
        for(int i=0;i<n.size();i++){
            m1=max(m1,n[i]);
            m2=min(m2,n[i]);
        }
        int c=0;
        for(int i=0;i<n.size();i++){
            if(n[i]<m1 && n[i]>m2){
                c++;
            }
        }
        return c;
    }
};

File: 2148-count-elements-with-strictly-smaller-and-greater-elements/2148-count-elements-with-strictly-smaller-and-greater-elements.py
class Solution:
    def countElements(self, nums: List[int]) -> int:
        c=0
        for i in range(len(nums)):
            f=[0,0]
            for j in range(len(nums)):
                if nums[j]<nums[i]:
                    f[0]+=1
                if nums[j]>nums[i]:
                    f[1]+=1
            if f[0] and f[1]:
                c+=1
        return c
        